lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -355,8 +356,9 @@ void SpecialFunctionHandler::handleCloseMerge(ExecutionState &state,
     llvm::errs() << "close merge: " << &state << " at " << i << '\n';
 
   if (state.openMergeStack.empty()) {
-    klee_warning(std::to_string(&state) + " ran into a close at " + std::to_string(i)
-                 + " without a preceding open\n");
+    std::ostringstream warning;
```

Comment:
This is minor, but I actually preferred to simply use `klee_warning` as you did before, instead of using an extra `std::ostringstream`